### DISCUSSION TITLE
Merci :)

### DISCUSSION THREAD

**Discussion by Myst521** [marian-m12l/studio#168] [General]
J'ai reçu ce matin sur whatsapp des audio de ma mère lisant une histoire pour ma fille et je suis tout de suite (et avec peu d'espoir) partit en quête d'un moyen de les mettre sur notre Lunii (v2.0.92). 
A ma grande surprise je suis vite arrivé ici et tout a super bien fonctionné avec la version 0.3.0 (peut être faudrait-il mettre un peu en avant et expliquer le FS_MOUNT c'est le seul point de friction que j'ai rencontré, mais je doit avouer être du genre à lire les instruction en diagonale ;) 

Bref tout ça pour dire : Merci beaucoup pour ce formidable outil, ergonomique et efficace, ma fille pourra ce soir écouter une histoire lu par sa grand mère avec photo et chapitrage et j'ai hâte de lui montrer :)

Je ne manquerais pas de venir signaler si je rencontre un problème. Bonne continuation et encore merci (3 fois ne sont pas de trop) !

**Comment by marian-m12l** [comment]
@Myst521 Merci pour le retour, content que l'outil soit utile :-) Je suis bien d'accord pour les instructions, elles seront à améliorer et clarifier pour la release de la 0.3.0

**Comment by jersou** [comment]
Je profite de cette discussion pour dire aussi : Merci beaucoup ! Studio est très bien fait et permet à pas mal d'enfants de profiter de bcp + d'histoires ! Et de ne plus tourner en rond sur les mêmes pack :)

**Reply by arthurlevy** [reply]
Merci beaucoup.
Des podcasts espagnol (d'Amerique) et les podcast France inter sur la Lunii de mes enfants bilingues.
Ils se sont bien amusés à dicter les titres pour les menus...
Tout marche au poil.
